src/zrc/sync/signals.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.db.models.signals import post_delete, post_save
+from django.dispatch import receiver
+from django.urls import reverse
+
+import requests
+from vng_api_common.models import APICredential
+from vng_api_common.utils import get_uuid_from_path
+from zds_client import Client, extract_params, get_operation_url
+
+from zrc.datamodel.models import ZaakInformatieObject
+
+logger = logging.getLogger(__name__)
+
+
+class SyncError(Exception):
+    pass
+
+
+def sync_create(relation: ZaakInformatieObject):
+    operation = 'create'
+
+    # build the URL of the Zaak
+    path = reverse('zaak-detail', kwargs={
+        'version': settings.REST_FRAMEWORK['DEFAULT_VERSION'],
+        'uuid': relation.zaak.uuid,
+    })
+    domain = Site.objects.get_current().domain
+    protocol = 'https' if settings.IS_HTTPS else 'http'
+    zaak_url = f'{protocol}://{domain}{path}'
+
+    logger.info("Zaak: %s", zaak_url)
+    logger.info("Informatieobject: %s", relation.informatieobject)
+
+    # Define the remote resource with which we need to interact
+    resource = 'objectinformatieobject'
+    client = Client.from_url(relation.informatieobject)
+
+    # TODO?
```

Comment:
what's the TODO here?

Reply:
I don't even know why I added it, probably just forgot to remove it along the way